src/caikit_nlp_client/grpc_client_introspection.py

```diff
@@ -89,7 +89,7 @@ def generate_text(self, model_id: str, text: str, **kwargs) -> str:
             log.error(f"Caught exception {exc}, re-throwing")
             raise exc
 
-    def generate_text_stream(self, model_id: str, text: str, **kwargs) -> [str]:
+    def generate_text_stream(self, model_id: str, text: str, **kwargs) -> list[str]:
```

Comment:
This is invalid in python 3.8 (`TypeError: 'type' object is not subscriptable`), so, depending on what version we want to target, we might want to change this to `List[str]` (making sure to add `from typing import List`)

Reply:
I have tried the suggestion. But it seems that `ruff -format`` is modifing this line to look like this. Is there away to disable this behaviour ?

Reply:
Let's drop 3.8.

We can set the target version in `ruff` using `target-version` in the `[too.ruff]` section, see [`target-version`](https://docs.astral.sh/ruff/settings/#target-version) docs